src/CommandBlock.js

```diff
@@ -17,14 +17,14 @@ import { ReactComponent as Right45 } from './svg/Right45.svg';
 import { ReactComponent as Right90 } from './svg/Right90.svg';
 import { ReactComponent as Right180 } from './svg/Right180.svg';
 import { ReactComponent as Loop } from './svg/Loop.svg';
-import { ReactComponent as LoopStart } from './svg/LoopStart.svg';
-import { ReactComponent as LoopEnd } from './svg/LoopEnd.svg';
 
 type CommandBlockProps = {
     commandName: string,
-    onClick: (evt: SyntheticEvent<HTMLButtonElement>) => void,
     disabled: boolean,
-    className?: string
+    looplabel?: string,
+    loopiterationsleft?: ?number,
```

Comment:
Please update the names of the new props to use camelCase: `loopLabel` and `loopIterationsLeft`